<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>73</x>
          <y>9</y>
          <width>62</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>NEURONAS</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="mallaMostrar">
        <property name="geometry">
         <rect>
          <x>233</x>
          <y>9</y>
          <width>541</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>40</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="datoId">
        <property name="geometry">
         <rect>
          <x>94</x>
          <y>40</y>
          <width>133</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>68</y>
          <width>38</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Voltaje:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="datoVoltaje">
        <property name="geometry">
         <rect>
          <x>94</x>
          <y>68</y>
          <width>133</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>96</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Posición X:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="datoPosicionX">
        <property name="geometry">
         <rect>
          <x>94</x>
          <y>96</y>
          <width>133</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>124</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Posición Y:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="datoPosicionY">
        <property name="geometry">
         <rect>
          <x>94</x>
          <y>124</y>
          <width>133</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>152</y>
          <width>23</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Red:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="datoRed">
        <property name="geometry">
         <rect>
          <x>94</x>
          <y>152</y>
          <width>133</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>180</y>
          <width>34</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Green:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="datoGreen">
        <property name="geometry">
         <rect>
          <x>94</x>
          <y>180</y>
          <width>133</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>208</y>
          <width>26</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Blue:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="datoBlue">
        <property name="geometry">
         <rect>
          <x>94</x>
          <y>208</y>
          <width>133</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="mostrarNeuronas">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>230</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Mostrar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="insertarInicio">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>236</y>
          <width>101</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Insertar Inicio</string>
        </property>
       </widget>
       <widget class="QPushButton" name="insertarFinal">
        <property name="geometry">
         <rect>
          <x>114</x>
          <y>236</y>
          <width>111</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Insertar Final</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sort">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>230</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Ordenar por ID</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sortV">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>230</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Ordenar por Voltaje</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Mostrar</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>10</y>
          <width>521</width>
          <height>231</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="BotonMostrardelatabla">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Mostrar de la tabla</string>
        </property>
       </widget>
       <widget class="QPushButton" name="BotonBuscardelatabla">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Buscar por ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="idBusqueda">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>210</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string> ID</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="BotonMostrarSortID">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Clasificar por ID</string>
        </property>
       </widget>
       <widget class="QPushButton" name="BotonMostrarSortV">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Clasificar por Voltaje</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Gráfica</string>
       </attribute>
       <widget class="QPushButton" name="dibujar">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Dibujar neuronas</string>
        </property>
       </widget>
       <widget class="QPushButton" name="limpiar">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Limpiar escena</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>185</x>
          <y>10</y>
          <width>591</width>
          <height>381</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="busqID">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>49</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnBusqID">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Mostrar Coordenadas (x,y)</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="mallaGrafNeuronas">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>161</width>
          <height>131</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="bruteForceGraphic">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>340</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Fuerza Bruta</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
